src/instrument.ts

```diff
@@ -50,8 +52,12 @@ interface observabilityOptions {
 
 export type {observabilityOptions as ObservabilityOptions};
 
-const TRACER_NAME = 'cloud.google.com/nodejs/spanner';
-const TRACER_VERSION = '7.14.0'; // Manually hard coded, TODO: remove
+const LIB_FQNAME = 'cloud.google.com/nodejs/spanner';
```

Comment:
I don't think we need to maintain two separate variables for `LIB_FQNAME` and `LIB_VERSION`. 

We can reuse `TRACER_NAME` and `TRACER_VERSION`